callbacks.js

```diff
@@ -1,38 +1,37 @@
-function isVowel (value) {
-    vowels = 'aeiouAEIOU'
-    // if the value is a single letter string and a vowel
-    if(value.length == 1 && vowels.includes(value) ){
-        return true
-    } else {
-        return false
+    function isVowel (value) {
+        vowels = 'aeiouAEIOU'
```

Comment:
You could just make the received value lower case, or use regular expressions.

Also use const when defining a constant.
```js
const vowels = 'aeiou';